// itptit-test-generator/dist/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.TestCase = exports.NumberArray = exports.Loop = exports.UseSubGeneration = exports.Int = exports.createTestCase = exports.generate = void 0;
const test_case_generator_1 = __importDefault(require("./test-case-generator"));
exports.createTestCase = test_case_generator_1.default;
const input_generator_1 = require("./input-generator");
Object.defineProperty(exports, "generate", { enumerable: true, get: function () { return input_generator_1.generate; } });
const decorators_1 = require("./src/decorators");
Object.defineProperty(exports, "Int", { enumerable: true, get: function () { return decorators_1.Int; } });
Object.defineProperty(exports, "UseSubGeneration", { enumerable: true, get: function () { return decorators_1.UseSubGeneration; } });
Object.defineProperty(exports, "Loop", { enumerable: true, get: function () { return decorators_1.Loop; } });
Object.defineProperty(exports, "NumberArray", { enumerable: true, get: function () { return decorators_1.NumberArray; } });
const input_test_case_1 = require("./src/classes/input-test-case");
Object.defineProperty(exports, "TestCase", { enumerable: true, get: function () { return input_test_case_1.TestCase; } });
